python/README.md

```diff
@@ -10,12 +10,12 @@ To install MXNet Python package, visit MXNet [Install Instruction](http://mxnet.
 
 For running unit tests, you will need the [nose PyPi package](https://pypi.python.org/pypi/nose). To install:
 ```bash
-pip install --upgrade nose
+sudo pip install --upgrade nose
```

Comment:
I believe this depends on the environment, and it's up to the user to decide whether a sudo is suitable or not.
For example, I may have pip that comes from virtualenv, conda, pyenv, or something else that requires only user access.

Reply:
+1 to @szha's comment.  For my environment for example adding sudo would install to the wrong python site-package directory.  I could see this as being unintuitive for many users.